Docs Update

AsnelChristian opened the pull request.

AsnelChristian committed:
Docs Update

Updates the documentation about the Smell and
Spelling aspects and their subaspects as the were modified,
and adds documentation about the Formatting aspect
that was added.

jayvdb reviewed (commented):
commit message

jayvdb reviewed (changes requested):
commit message

jayvdb commented on Root/Formatting/README.rst:
```diff
@@ -0,0 +1,46 @@
+* `Length <Length/README.rst>`_
+* `Quotation <Quotation/README.rst>`_
+* `Spacing <Spacing/README.rst>`_
+Example
```
blank line above this.

AsnelChristian replied:
this is automatically generated, do you want me to modify the function that generates it

jayvdb replied:
yes please.

AsnelChristian replied:
Done :+1:

AsnelChristian committed:
generate: Add newline after ``Index``

Adds a blank line after the first line containing the
links to the parent aspect and the index.

AsnelChristian commented:
Commit message updated
